release-notes/6.0/cve.md

```diff
@@ -2,11 +2,11 @@
 
```

Comment:
Should you also switch the title to say .NET 6 Common Vulnerabilities and Exposures (CVEs)? Or define the acronym at some point?

Reply:
Yes. I was thinking about that, too.

How about this: https://github.com/dotnet/core/pull/7506/commits/cf8f398bde88b97e554188c9346ad60e96079c22

Reply:
Looks good.